src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1722,14 +1790,28 @@ namespace cryptonote
           burnt_loki += get_burned_amount_from_tx_extra(tx.extra);
         }
       }
-      
+
       emission_amount += coinbase_amount - tx_fee_amount;
       total_fee_amount += tx_fee_amount;
+      if (cache_to && cache_to == height)
```

Comment:
`cache_to` can be set by when `count < CACHE_EXCLUSIVE` if you request the coinbase TX sum on 2 different threads on a block receipt boundary (1 before the block, 1 after the block arrive). We'd end up overwriting the cache with stale data.